### ISSUE TITLE
A11y_Model Builder_Add Data_Data Preview_Screen Reader : When Narrator is on data table is not accessed with any table shortcut keys.

### ISSUE THREAD

**Issue by manaswita-chougule** [dotnet/machinelearning-modelbuilder#2210]
### GitHubTags:
#A11yMAS; #A11yTCS; #A11ySev3; #DesktopApp; #Visual Studio Client; #SH-ModelBuilder-Win32-Jun2022; #Win32; #WCAG1.3.1; #Info and relationship; #Narrator; #Regressed:09-14-22; #Closed;

### Environment Details:
Application Name: Model Builder
Windows Version: Windows 11 22H2 OS Builder 25145.1011
Visual Studio: Microsoft Visual Studio Enterprise 2022 (64-bit) - Preview Version 17.3.0 Preview 2.0

### Repro Steps:

1. Turn on Narrator.
2. Install Visual studio and open Visual studio
3. Navigate to C# project name and right click on it
4. Navigate to "Add" button and press enter
5. Navigate to "Machine learning model" and press enter
6. "Add new item" popup will open
7. Navigate to "Add" button and press enter
8. "Select scenario" popup will open and select "Forecasting" scenario
9. "Select training environment" page will open then select "Next step"
10. "Add Data" page will appear browse and upload the sample data file.
11. tab till Data Preview section.
12. Tab till Table radio button and press enter.
13. Data Table will appear.
14. Check whether When Narrator is on data table is not accessed with any table shortcut keys or not.

### Actual Result:
When Narrator is on data table is not accessed with any table shortcut keys.

### Expected Result:
When Narrator is on data table is not accessed with any table shortcut keys.

![2188-Screen-reader](https://user-images.githubusercontent.com/91586101/182530844-139da3aa-bb6f-417d-bbd7-d5b53b4ae36d.gif)

**Comment by beccamc** [comment]
@manaswita-chougule What are table shortcut keys? Is there documentation on this requirement?

**Comment by beccamc** [comment]
@manaswita-chougule Ping on this. We have previously gotten feedback from the accessibility pass that this component should not be a tab stop because you cannot interact with it ([Issue 1131](https://github.com/dotnet/machinelearning-modelbuilder/issues/1131)). Instead it should read aloud in scan mode. That is the current behavior. 

If only reading in scan mode is not the correct behavior can you point me to the appropriate documentation so I can fix this properly and not need to change it again?

**Comment by manaswita-chougule** [comment]
> @manaswita-chougule What are table shortcut keys? Is there documentation on this requirement?

Table keys here I want to refer Alt+Ctrl+arrow keys.

**Comment by beccamc** [comment]
@manaswita-chougule Can you take another look at this please? In scan mode I am able to use the table keys. I cannot use the table keys until I am in the contents of the table. In the header it doesn't work. In the below gif you can see me jumping around once I reach the first row of numbers. 

![table](https://user-images.githubusercontent.com/9122518/189764972-de07c762-423d-4b44-b6e5-fde3dfb08254.gif)

I've tested other WPF apps and they behave the same way.

**Comment by manaswita-chougule** [comment]
Verified the issue in below environment:
Microsoft Visual Studio Enterprise 2022 (64-bit) - Preview
Version 17.4.0 Preview 1.0

The issue is fixed as with arrows will able to navigate through table headers and to data cells from headers. In data cells will able to navigate using table keys(Ctrl+Alt+arrows). Hence closing the bug. Please find below attachment for reference.

![2210_Fixed](https://user-images.githubusercontent.com/91586101/190153436-1f2806d4-0980-428b-ab00-5a0010ae594f.gif)
